ARK Server Manager/Lib/NetworkUtils.cs

```diff
@@ -122,6 +122,14 @@ public static NetworkAdapterEntry GetPreferredIP(IEnumerable<NetworkAdapterEntry
 
         public class AvailableVersion
         {
+            public AvailableVersion()
+            {
+                IsValid = false;
+                Current = new Version(0, 0);
+                Upcoming = new Version(0, 0);
+                UpcomingETA = "unknown";
```

Comment:
If this string would appear to end users, it should come from the strings file (so it can be localized)